src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -81,6 +81,8 @@ public User createUser(User newUser) {
     genderPreferences.add(Gender.OTHER);
     newUser.setGenderPreferences(genderPreferences);
     newUser.setMaxRange(10);
+    newUser.setLatitude(0);
+    newUser.setLongitude(0);
```

Comment:
Why set the position default as 0,0? wouldn't it be possible to get the position of the user when logging in/registering for instance?

Reply:
@JoMuelle What happens if the user blocks position?

Reply:
@TwoDigitsOneNumber It makes sense with the separate API-Endpoint (the separate api-endpoint might increase traffic, though), see https://github.com/sopra-fs22-group-07/client/pull/165#pullrequestreview-981004051

Reply:
i agree that a separate endpoint is the way to go. that's why i did it the way i did. there fore i set the default in (createUser) to 0 and 0, because it needs to be initialized so that the SQL database is happy (can't be null) and our functionality would also not work if it were null. E.g. retrieving the black card according to distance to other users if the user blocked access. Still it would return 0,0 but it's a bit of redundancy that keeps the system more stable.